src/components/application_manager/src/helpers/application_helper.cc

```diff
@@ -37,9 +37,9 @@ void DeleteCommands(ApplicationSharedPtr app, ApplicationManager& app_manager) {
 
 void DeleteSubmenus(ApplicationSharedPtr app, ApplicationManager& app_manager) {
   auto accessor = app->sub_menu_map();
-  const auto& sub_menu_map = accessor.GetData();
+  const auto sub_menu_map = accessor.GetData();
 
-  for (const auto& smenu : sub_menu_map) {
+  for (const auto smenu : sub_menu_map) {
```

Comment:
```suggestion
  for (const auto& smenu : sub_menu_map) {
```

Reply:
fixed in e6e5767ef10ee27415d93deeac04dd30739b2f6e